### DISCUSSION TITLE
Documentation?

### DISCUSSION THREAD

**Discussion by theAkito** [edumeet/edumeet#1090] [Q&A]
Cannot seem to find any documentation for adminstrators of their own selfhosted instance. The only thing I could find is [this manual from 2020](https://edumeet.org/wp-content/uploads/2020/11/eduMEET_v3_4-User_Manual.pdf). That's it.

Where is the complete documentation?
Where are all configuration options explained?

For example, I'm trying to figure out, how to set up authentication, how to enable claiming a room, etc....

Where can I read about everything important for administrators?

**Comment by zvonem** [comment]
I agree. We have used eduMeet 3.5 for last couple of years and would like to upgrade to eduMeet 4. So we would benefit from a single complete documentation with couple of usage examples (from single server installation, load balanced installation to multi server - multi tenant etc.). Also, is there no more LTI authentication?
